pgmpy/tests/test_models/test_BayesianModel.py

```diff
@@ -226,6 +226,21 @@ def test_remove_nodes_from(self):
         self.assertRaises(ValueError, self.G1.get_cpds, 'diff')
         self.assertRaises(ValueError, self.G1.get_cpds, 'grade')
 
+    def test__eq__(self):
+        G4 = BayesianModel([('a', 'd'), ('b', 'd'),
+                                ('d', 'e'), ('b', 'c')])
+        cpd_a = TabularCPD('a', 2, [[0.2], [0.8]])
+        cpd_b = TabularCPD('b', 2, [[0.3, 0.7], [0.7, 0.3]],
+                       evidence=['a'],
+                       evidence_card=[2])
+        cpd_c = TabularCPD('c', 2, [[0.3, 0.7], [0.7, 0.3]],
+                       evidence=['b'],
+                       evidence_card=[2])
+        G4.add_cpds(cpd_a, cpd_b, cpd_c)
+        G5 = G4.copy()
+        self.assertTrue(G4 == G5)
+        self.assertFalse(self.G1 == self.G)
```

Comment:
The tests need to cover more possible situations. For example different order of CPDs in the network, check on the networks with same structure but different CPDs etc.